examples/enterprise/variables.tf

```diff
@@ -31,9 +31,14 @@ variable "access_tags" {
   description = "List of access tags to apply to resources for IAM policy scoping."
 }
 
-
 variable "ibmcloud_api_key" {
   type        = string
   description = "IBM Cloud API key used for authentication."
   sensitive   = true
 }
+
+variable "ibmcloud_resource_controller_api_endpoint" {
```

Comment:
No need to expose this in the example - just hard code it in the terraform code. You can use public endpoint for the example